src/ShortidType.php

```diff
@@ -31,35 +36,47 @@ public function getSQLDeclaration(array $column, AbstractPlatform $platform): st
         return $platform->getVarcharTypeDeclarationSQL($field).' '.$platform->getColumnCollationDeclarationSQL('utf8_bin');
     }
 
-    public function convertToPHPValue($value, AbstractPlatform $platform)
+    /**
+     * {@inheritdoc}
+     */
+    public function convertToPHPValue($value, AbstractPlatform $platform): ?Shortid
     {
         if (empty($value)) {
-            return;
+            return null;
```

Comment:
again, it doesn't make sense to me

Reply:
A return type definition like `void|ShortId` is not allowed, so by adding return types this must be changed accordingly:

"Fatal error:  Void can only be used as a standalone type"

Additionally from my point of view sometimes returning nothing from a method that is named `convertToWhatever()` is also quite surprising.